Adyen.podspec

```diff
@@ -15,7 +15,7 @@ Pod::Spec.new do |s|
   s.swift_version = '5.7'
   s.frameworks = 'Foundation'
   s.default_subspecs = 'Core', 'Components', 'Actions', 'Card', 'Encryption', 'DropIn', 'Session'
-  s.pod_target_xcconfig = {'SWIFT_SUPPRESS_WARNINGS' => 'YES', 'BUILD_LIBRARY_FOR_DISTRIBUTION' => 'YES' }
+  s.pod_target_xcconfig = {'SWIFT_SUPPRESS_WARNINGS' => 'YES' }
```

Comment:
Why are we removing `BUILD_LIBRARY_FOR_DISTRIBUTION' => 'YES` ?

Reply:
It was added during my experimentations with cocoapods in the target branch without a PR. This line is just reverting things to as they were before. Sorry for confusion!